tests/settings.py

```diff
@@ -22,6 +22,7 @@
     'django.contrib.sites',
     # admin
     'django.contrib.admin',
+    'admin_auto_filters',
```

Comment:
let's not forget to update https://github.com/openwisp/openwisp-utils#using-the-admin_theme